flixel/FlxObject.hx

```diff
@@ -826,6 +826,11 @@ class FlxObject extends FlxBasic
 		return point.subtract(Camera.scroll.x * scrollFactor.x, Camera.scroll.y * scrollFactor.y);
 	}
 	
+	public inline function getPosition():FlxPoint
```

Comment:
I'd prefer to structure this code similar to getMidPoint (with optional parameter), That way developers can easily optimize code to avoid allocations or pooling costs.